Attendee.test.js

```diff
@@ -0,0 +1,30 @@
+import { Attendee } from './Attendee';
+
+test('Check for correct value of screenName', () => {
+    const attendee = new Attendee();
+    const expectedScreenName = "Jaz";
+    attendee.setScreenName_(expectedScreenName);
+    const gotScreenName = attendee.getScreenName();
+    expect(gotScreenName).toBe(expectedScreenName);
+}); 
+test('Check for correct value of sessionId', () => {
+    const attendee = new Attendee(); 
+    const expectedId = "JTK614";
+    attendee.setSessionId_(expectedId);
+    const gotId = attendee.getSessionId();
+    expect(gotId).toBe(attendee.expectedId);
```

Comment:
you want to expect the attendee.getSessionId() to be what you set it as (expectedId)